### PR TITLE
fix(`general`): get ci green on 6219

### PR DESCRIPTION
**Pull request by Evalir**

### CODE REVIEW

File: crates/common/src/lib.rs [hunk 1]
```diff
@@ -28,6 +28,7 @@ pub mod term;
 pub mod traits;
 pub mod transactions;
 pub mod types;
+pub mod units;
```
Comment (mattsse) [inline crates/common/src/lib.rs:31]: I don't see a units.rs in this PR?
does this simply add an existing mod?

File: crates/cast/src/lib.rs [hunk 2]
```diff
@@ -1241,11 +1241,33 @@ impl SimpleCast {
...
+                .trim_end_matches(".000000000")
+                .to_string(),
+            "mwei" | "mega" | "megaether" => foundry_common::units::format_units(value, 6)?
+                .trim_end_matches(".000000")
+                .to_string(),
+            "kwei" | "kilo" | "kiloether" => {
+                foundry_common::units::format_units(value, 3)?.trim_end_matches(".000").to_string()
+            }
+            "wei" => {
+                foundry_common::units::format_units(value, 0)?.trim_end_matches(".0").to_string()
```
Comment (mattsse) [inline crates/cast/src/lib.rs:1267]: why do we need all of this now?

**Review by mattsse — CHANGES_REQUESTED** [review]
pending @DaniPopes

File: crates/common/src/lib.rs (hunk 1 above)
Comment (Evalir) [reply crates/common/src/lib.rs:31]: correct

**Comment by DaniPopes** [comment]
I had made those changes to migrate to alloy parse_units. The current impl on master is not good.

**Comment by Evalir** [comment]
@DaniPopes , yep, reverted locally. I still want this PR open to bundle all the green ci changes for 6219.

**Review by DaniPopes — APPROVED** [review]

**Pull request merged**